node_modules/oae-principals/lib/rest.user.js

```diff
@@ -471,6 +479,78 @@ OAE.globalAdminRouter.on('post', '/api/user/:userId', _handleUpdateUser);
 OAE.tenantRouter.on('post', '/api/user/:userId', _handleUpdateUser);
 
 /**
+ * @REST postUserEmailVerify
+ *
+ * Verify a guest token
+ *
+ * @Server      tenant
+ * @Method      POST
+ * @Path        /user/guest/verify
+ * @FormParam   {string}        token                       The token to verify
+ * @FormParam   {string}        username                    The username for the guest
+ * @FormParam   {string}        password                    The password for the guest
+ * @HttpResponse                200                         The token was valid and the login credentials were associated with the user account
+ * @HttpResponse                400                         A token, username or password was not specified
+ * @HttpResponse                404                         The token was not associated with a user
+ */
+OAE.tenantRouter.on('post', '/api/user/guest/verify', function(req, res) {
+    PrincipalsAPI.verifyGuestToken(req.ctx, req.body.token, req.body.username, req.body.password, function(err) {
+        if (err) {
+            return res.send(err.code, err.msg);
+        }
+
+        res.send(200);
```

Comment:
Should verifying the token automatically sign in the user and redirect to the /me page ?